app/views/rooms/show.html.erb

```diff
@@ -39,7 +38,27 @@
       </tbody>
     </table>
   </div>
-  <td><%= react_component('ScheduleEventForm', userID: 1, roomID: room.id ) %></td>
+
+<%= form_for Event.new do |f| %>
+
+<div class="field">
+  <%= f.label :start_time %><br>
+  <%= f.datetime_select :start_time %>
+</div>
+<div class="field">
+  <%= f.label :duration %><br>
+  <%= f.number_field :duration %>
 </div>
+<div class="field">
+  <%= f.label :description %><br>
+  <%= f.text_field :description %>
+</div>
+<div class="actions">
+  <%= f.submit %>
+</div>
+<%= f.hidden_field(:user_id, value: current_user.id) %>
```

Comment:
Remember that we can handle assigning the `:user_id` inside of the controller and we don't have to do it in the view.